meerk40t/core/svg_io.py

```diff
@@ -401,7 +398,7 @@ def _write_operation(xml_tree, node):
         @param node:
         @return:
         """
-        subelement = SubElement(xml_tree, "operation")
+        subelement = SubElement(xml_tree, MEERK40T_XMLS_ID + ":operation")
```

Comment:
I'm not sure on the backwards compatibility here. While strictly speaking this better adheres to the various RFCs it might be somewhat breaking.

Reply:
The load routine, should have it covered